ledger/participant-integration-api/src/main/scala/platform/indexer/ha/HaCoordinator.scala

```diff
@@ -58,6 +58,7 @@ case class HaConfig(
     workerLockAcquireRetryMillis: Long = 500,
     workerLockAcquireMaxRetry: Long = 1000,
     mainLockCheckerPeriodMillis: Long = 1000,
+    mainLockCheckerJdbcNetworkTimeoutMillis: Int = 10000,
```

Comment:
Where does the default come from?

Reply:
This is a T-Shirt number, with aligning the network timeout on the client side to the keepalives timeout on the server side. The latter is 15 seconds, and this one will fail after 10 seconds of inactivity, but it needs an additional 10 seconds, if the socket is tunneled through SSL (usually the case), so client will continue after 20 seconds in this case.
@tudor-da @simonmaxen-da @mziolekda WDYT?

Reply:
The server in this context is the Postgres server?

Reply:
We only have keepalives for PostgreSQL.

Reply:
I think 20 seconds instead of 10 doesn't change the landscape here. It is still responsive